ucm2/sof-soundwire/rt715-sdca.conf

```diff
@@ -0,0 +1,20 @@
+# Use case Configuration for sof-soundwire card
+
+SectionDevice."Mic" {
+	Comment	"SoundWire microphones"
+
+	EnableSequence [
+		cset "name='rt714 FU02 Capture Switch' 1"
+	]
+
+	DisableSequence [
+		cset "name='rt714 FU02 Capture Switch' 0"
+	]
+
+	Value {
+	      CapturePriority 100
+	      CapturePCM "hw:${CardId},4"
+	      CaptureSwitch "rt714 FU02 Capture Switch"
+	      CaptureVolume "rt714 FU02 Capture Volume"
```

Comment:
Missing MixerElem for PA.

Reply:
For the MixerElem, is `CaptureMixerElem "rt714 FU02"` good enough?
I checked how to set control for MixerElem in alsa-lib. It seems very little information about it. I don't know what is the best solution to set the MixerElem. Could you give a little hints?